### PR TITLE
Fetches prebuilts from Azure cache exclusively for bsb projects

### PR DESCRIPTION
**Pull request by prometheansacrifice**

### CODE REVIEW

**Comment by prometheansacrifice** [comment]
We need atdgen

File: src/LSP.re [hunk 1]
```diff
@@ -150,15 +163,283 @@ module Server = {
...
+                                     let filter = "deletedFilter=excludeDeleted&statusFilter=completed&resultFilter=succeeded";
+                                     let latest = "queryOrder=finishTimeDescending&$top=1";
+                                     Js.(
+                                       Promise.(
+                                         Https.getCompleteResponse(
+                                           {j|$restBase/$proj/_apis/build/builds?$filter&$master&$latest&api-version=4.1|j},
+                                         )
+                                         |> then_(
+                                              bindResultAndResolvePromise(
+                                                responseText =>
```
Comment (prometheansacrifice) [inline src/LSP.re]: Apologies for parsing by hand. I need to fix this with atdgen

**Comment by prometheansacrifice** [comment]
@rusty-key Would you like to review this?

**Comment by prometheansacrifice** [comment]
TODO: handle network failures
TODO: Possible to open command shell and run esy there?

**Comment by prometheansacrifice** [comment]
In a much better state to be reviewed

**Comment by rusty-key** [comment]
> @rusty-key Would you like to review this?

Sorry, just returned from vacation. If it makes sense, I can review retroactively tomorrow

**Pull request merged**